### ISSUE TITLE
Runescape

### ISSUE THREAD

**Issue by JoshuaFern** [luxtorpeda-dev/packages#303]
* Project link: https://content.runescape.com/downloads/ubuntu/pool/non-free/r/runescape-launcher/runescape-launcher_2.2.9_amd64.deb
* Project license: Proprietary
* Does it work on 64-bit only Linux: Yes
* Is source code available?: No
* List of Steam AppIds of supported games:
    - https://steamdb.info/app/1343400/

## Additional information

Inspired by the recent addition of RuneLite for OSRS, this is the native Linux client for modern Runescape.

Runescape on Steam is Windows-only despite having this native Linux release.

Like OSRS, Runescape for Steam has a unique feature to login using your Steam account. This feature is unimplemented in this version so you would need to login using a typical email/password. See: https://support.runescape.com/hc/en-gb/articles/360013472137-Add-a-login-email-to-a-Steam-created-account

**Comment by d10sfan** [comment]
Is there a way to get the launcher without a deb, like an archive or an appimage?

For systems without ubuntu. Not sure if normal tar type tools could extract it.

**Comment by JoshuaFern** [comment]
I'm not aware of any other releases
The deb needs to be extracted with `ar x runescape.deb` the ar utility is included in the flatpak.

**Comment by d10sfan** [comment]
@JoshuaFern Do you have the tar utility in flatpak? It should be on normal distros, but wondering about flatpak.

**Comment by JoshuaFern** [comment]
Yep, I bet many of these utilities are also included in the Steam runtime.
```
nixos-workstation% find . | grep bin/tar
./.local/share/Steam/steamapps/common/SteamLinuxRuntime/var/scout_0.20210126.2/files/bin/tar
./.local/share/Steam/steamapps/common/SteamLinuxRuntime_soldier/var/soldier_0.20210208.0/files/bin/tar
./.local/share/Steam/steamapps/common/SteamLinuxRuntime_soldier/var/soldier_0.20210208.0/files/sbin/tarcat
```
I don't see `ar` in the runtime though.

**Comment by d10sfan** [comment]
Added in https://github.com/luxtorpeda-dev/packages/commit/e5381b818f649648bac5067ae2b173bdefe01d77

Tested to the point of the launcher downloading.

**Comment by JoshuaFern** [comment]
Seems to be working fine, cheers.
